metaflow/plugins/kfp/constants.py

```diff
@@ -14,4 +14,5 @@
 
 # TODO: Switch out of using this role
 S3_BUCKET = "s3://workspace-zillow-analytics-stage/aip/metaflow"
-S3_AWS_ARN = "arn:aws:iam::170606514770:role/dev-zestimate-role"
\ No newline at end of file
+S3_AWS_ARN = "arn:aws:iam::170606514770:role/dev-zestimate-role"
+S3_AWS_REGION = "us-west-2"
```

Comment:
What is the difference between `S3_AWS_REGION` and `METAFLOW_AWS_S3_REGION`? That'd be a good comment.  Same for the others, such as ARN.

Why not use `METAFLOW_AWS_ARN` and `METAFLOW_AWS_S3_REGION` and `DATASTORE_SYSROOT_S3`?

Reply:
Removed these. No longer needed.